CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Fixed `mergeComponentMetadata` function to get defaultProps and defaultPropValues of components.
```

Comment:
```suggestion
* Fixed controls to take default prop values from `defaultPropValues` since `defaultProps` in function component has been deprecated.
```

Reply:
It's updated.